target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8070-eap620hd-v1.dts

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR BSD-2-Clause
+
+/dts-v1/;
+
+#include "ipq8074.dtsi"
+#include "ipq8074-hk-cpu.dtsi"
```

Comment:
```suggestion
#include "ipq8074-ac-cpu.dtsi"
```

If its IPQ8070A really, check SMEM ID for it

Reply:
Turns out it's actually an IPQ8072a, just like the EAP660HDv1

Reply:
Even better, I see you already updated the commit description.